scripts/scraper/venv/lib/python3.12/site-packages/_pytest/_code/code.py

```
@@ -0,0 +1,1567 @@
+
+    @property
+    def path(self) -> Path | str:
+        """Return a path object pointing to source code, or an ``str`` in
+        case of ``OSError`` / non-existing file."""
+        if not self.raw.co_filename:
+            return ""
+        try:
+            p = absolutepath(self.raw.co_filename)
+            # maybe don't try this checking
+            if not p.exists():
+                raise OSError("path check failed.")
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)